stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/home/HomeViewModel.kt

```diff
@@ -92,11 +92,15 @@ class HomeViewModel(
         when (action) {
             is UiAction.LogoutClicked -> {
                 viewModelScope.launch {
-                    chatClient.disconnect(false).await()
+                    ChatClient.instance().disconnect(false).await()
```

Comment:
Any reason why we don't just use the chatClient like before?

Reply:
No, let me revert this.